tests/integration/session/test_base.py

```diff
@@ -280,3 +282,15 @@ async def test_task_cancellation_cancels_query(config, test_session):
         task.cancel()
         await task
     await check_table_does_not_exist_or_empty(session, "job_cancel_test_output")
+
+
+@pytest.mark.parametrize(
+    "source_type", ["spark", "databricks_unity", "snowflake", "bigquery"], indirect=True
+)
```

Comment:
We could remove this `parametrize` since these are the default source types to test.